middleware/service/source/manager/handle.cpp

```diff
@@ -162,7 +163,7 @@ namespace casual
                // send event, at least domain-manager want's to know...
                common::message::event::process::Assassination event{ common::process::handle()};
                event.target = entry.target;
-               event.contract = entry.service->timeout.contract;
+               event.contract = entry.service->timeout.contract.value_or( common::service::execution::timeout::contract::Type::linger);
```

Comment:
Do we take global into account during transformation of the configuration? It seems like there should be code changes for that? I mean, if duration is specified but not contract for a service, then the global contract should be used.